src/crimson/os/seastore/random_block_manager/nvme_manager.cc

```diff
@@ -253,20 +255,21 @@ NVMeManager::allocate_ertr::future<> NVMeManager::alloc_extent(
    *
    */
   return find_free_block(t, size
-      ).safe_then([] (auto alloc_extent) mutable
-	-> allocate_ertr::future<> {
+      ).safe_then([&t, this] (auto alloc_extent) mutable
+	-> allocate_ret {
 	logger().debug("after find_free_block: allocated {}", alloc_extent);
 	if (!alloc_extent.empty()) {
 	  rbm_alloc_delta_t alloc_info {
-	    extent_types_t::RBM_ALLOC_INFO,
 	    alloc_extent,
 	    rbm_alloc_delta_t::op_types_t::SET
 	  };
-	  // TODO: add alloc info to delta
+	  t.add_rbm_alloc_info_blocks(alloc_info);
```

Comment:
I think I just forgot this from before.  How do we ensure that multiple concurrent transactions don't allocate the same block (or detect the conflict if they do?)